src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/ReportController.kt

```diff
@@ -38,7 +38,12 @@ class ReportController(
     fun getByNameAndVersion(): ReportVersionPageViewModel
     {
         val reportName = context.params(":name")
-        val version = context.params(":version")
+        val version: String
+        if (context.params()[":version"].isNullOrEmpty()) {
+            version = reportRepository.getLatestVersion(reportName).id
+        } else {
+            version = context.params(":version")
+        }
```

Comment:
Could do something like this for brevity:
```suggestion
        val version = context.params().getOrElse(":version") { reportRepository.getLatestVersion(reportName).id }!!
```